cmd/humioctl/views_update.go

```diff
@@ -0,0 +1,52 @@
+// Copyright © 2020 Humio Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+"fmt"
+
+"github.com/spf13/cobra"
```

Comment:
There is missing identation here. Are you missing a formatting hook in goland?

Reply:
I fixed the indentation. I have not changed anything in Goland yet. I will check for options to automatically format the code.